Makefile

```diff
@@ -970,6 +975,9 @@ $(SETUP_ENVTEST_BIN): $(SETUP_ENVTEST) ## Build a local copy of setup-envtest.
 .PHONY: $(KPROMO_BIN)
 $(KPROMO_BIN): $(KPROMO) ## Build a local copy of kpromo
 
+.PHONY: $(YQ_BIN)
```

Comment:
I'm still getting 

```
➜  cluster-api git:(pr/6681) ./hack/get-project-maintainers.sh
make: *** No rule to make target `hack/tools/bin/yq'.  Stop.
```

Probably because the makefile was refactored and the script runs `make hack/tools/bin/yq` but this just has a `make yq` target

Reply:
if you change /hack/get-project-maintainers.sh to

```
YQ="${REPO_ROOT}/hack/tools/bin/yq"
cd "${REPO_ROOT}" && make "${YQ##*/}" &>/dev/null
```
it should work

Reply:
fixed, it worked because first I called make yq, then the binary was already there when testing the script